io.github.Prt.Sebicodes99.yml

```diff
@@ -0,0 +1,19 @@
+app-id: io.github.Prt.Sebicodes99
```

Comment:
As I said, you are not user `Prt`.

Reply:
Oh sorry, I tought it was the lowercase one, I will fix it